test/unit/mdc-select/mdc-select.test.js

```diff
@@ -527,3 +553,9 @@ test('keydown is not added to the native select when initialized', () => {
   td.verify(component.foundation_.handleKeydown(td.matchers.anything()), {times: 0});
   document.body.removeChild(fixture);
 });
+
+test('#constructor instantiates a leading icon if an icon element is present', () => {
```

Comment:
One question out of curiosity: coverage is telling me we're covering both the if and else case in `initialize` where it conditionally instantiates leading icon... but are we specifically testing the `else`, given that all the fixtures have an icon?